src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/UserPersonDetail.kt

```diff
@@ -29,7 +30,7 @@ data class UserPersonDetail(
   val roles: List<UserCaseloadRole> = listOf(),
 
   @OneToMany(fetch = FetchType.LAZY, cascade = [CascadeType.ALL], mappedBy = "user")
-  val caseloads: List<UserCaseload> = listOf(),
+  var caseloads: List<UserCaseload> = listOf(),
```

Comment:
I wonder if this could just be `= mutableListOf()` and keep it has a val?

Reply:
Do you think that is better?